tests/ctst/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "artesca-ctst",
+  "version": "1.0.0",
+  "description": "Testing Artesca features with CTST",
+  "main": "index.js",
+  "repository": "git+https://github.com/scality/artesca.git",
+  "author": "Scality",
+  "license": "ISC",
+  "private": true,
+  "scripts": {
+    "setup": "PREPARE=1 ./run.sh",
+    "dry-run": "./run.sh all --dry-run",
+    "premerge": "./run.sh premerge",
+    "only": "./run.sh only",
+    "postmerge": "./run.sh all"
```

```suggestion
    "setup": "PREPARE=1 ./run.sh",
    "dry-run": "./run.sh all --dry-run",
    "premerge": "./run.sh premerge",
    "only": "./run.sh only",
```

We don't have any post merge in Zenko.

Also, do you think we could document how to run these CTST tests locally, considering we have a Zenko running locally or in a VM?

---

We don't actually use any of those commands, so i'll remove them all